frontend/modules/shared/data-access/src/lib/services/ticket.service.ts

```diff
@@ -42,4 +45,8 @@ export class TicketService extends AbstractApiService<TicketWithResponses> {
       headers: { 'Content-Type': 'application/json' },
     });
   }
+
+  createTicket(newTicket: CreateTicketDto) {
+    return this._http.post(`${this._resourceUrl}`, newTicket);
```

Comment:
If there's no further logic, you can make `TikcetService` to extend `AbstractApiService<Ticket, CreateTicketDto>`. The 2nd generic parameter is the POST create request. Then the `create` method will do the same.

Reply:
Did not know about that :D, changed